platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -267,6 +283,31 @@ public void createNextExperiment() {
         }
     }
 
+    private void createRabbitMQ(ExperimentConfiguration config) throws Exception {
+        String rabbitMQAddress = EnvVariables.getString(RABBIT_MQ_EXPERIMENTS_HOST_NAME_KEY, (String)null);
+        if (rabbitMQAddress == null) {
+            LOGGER.info("Starting new RabbitMQ for the experiment...");
+            rabbitMQAddress = controller.containerManager.startContainer(EnvVariables.getString(RABBIT_IMAGE_ENV_KEY),
+                    Constants.CONTAINER_TYPE_BENCHMARK, null,
+                    new String[] {  },
+                    null, null, config.id);
+            if (rabbitMQAddress == null) {
+                experimentStatus.addError(HobbitErrors.UnexpectedError); // FIXME
+                throw new Exception("Couldn't start new RabbitMQ for the experiment");
+            }
+
+            experimentStatus.setRootContainer(rabbitMQAddress);
+            LOGGER.info("Using the newly started RabbitMQ for the experiment: {}", rabbitMQAddress);
+        } else {
+            LOGGER.info("Using the configured RabbitMQ for the experiment: {}", rabbitMQAddress);
+        }
+        experimentStatus.setRabbitMQContainer(rabbitMQAddress);
+
+        RabbitMQConnector rabbitMQConnector = new RabbitMQConnector(controller, experimentStatus.getRabbitMQContainer());
+        controller.setExpRabbitMQConnector(rabbitMQConnector);
```

Comment:
Maybe we should switch to always rely on the rabbit container of the experimentStatus object instead of having this setter... :thinking: